### PR TITLE
Use light-my-request types instead of duplicating typings. fixes #2098

### LINKED ISSUE #2098: The HTTPInjectOptions typescript interface does not define the 'cookies' property
## 🐛 Bug Report

The HTTPInjectOptions typescript interface should define the `cookies` property

## To Reproduce

The following code can not be compiled with typescript (sorry, the code cannot be run as is, but describes the problem well):
```typescript
    const fastify: FastifyInstance = ...
    const response: HTTPInjectResponse = await fastify
        .inject({
            method: 'POST',
            url: '/some-url',
            payload: { 'some': 'payload' },
            cookies: { foo: 'bar' },
        })
```

## Expected behavior

The code should be compilable with typescript, but I get the following error:

```
error TS2345: Argument of type '{ method: "POST"; url: string; payload: {... }; cookies: { foo: string; }; }' is not assignable to parameter of type 'string | HTTPInjectOptions'.
  Object literal may only specify known properties, and 'cookies' does not exist in type 'HTTPInjectOptions'.
```

## Your Environment

- *node version*: 12
- *fastify version*: >=2.12.0
- *os*: Mac

### PR DESCRIPTION
**Pull request by gyszalai**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by gyszalai** [comment]
> Would it be possible to avoid the breaking change for typescript users? I mean we should have `HTTPInjectOptions` and `HTTPInjectResponse` inherit from the same types of light-my-request?

Is it okay if these interfaces are imported with aliases like this:
```ts
import { InjectOptions as HTTPInjectOptions, Response as HTTPInjectResponse } from 'light-my-request'
```
or extending these interfaces are necessary like this:
```ts
interface HTTPInjectOptions extends LightMyRequest.InjectOptions {}
interface HTTPInjectResponse extends LightMyRequest.Response {}
```

**Comment by mcollina** [comment]
I'd go for the second option.

**Comment by gyszalai** [comment]
done

File: test/types/index.ts [hunk 1]
```diff
@@ -609,13 +610,13 @@ server.listen({
...
 // http injections
-server.inject({ url: '/test' }, (err: Error, res: fastify.HTTPInjectResponse) => {
+server.inject({ url: '/test' }, (err: Error, res: LightMyRequest.Response) => {
   server.log.debug(err)
   server.log.debug(res.payload)
 })
 
 server.inject({ url: '/testAgain' })
-  .then((res: fastify.HTTPInjectResponse) => console.log(res.payload))
+  .then((res: LightMyRequest.Response) => console.log(res.payload))
```
Comment (mcollina) [inline test/types/index.ts]: Can you add another test with the old types as well?

Comment (gyszalai) [reply test/types/index.ts]: oops, I forgot it, sorry.

**Pull request merged**